test/plots/band-clip.js

```diff
@@ -14,3 +15,23 @@ export default async function () {
     ]
   });
 }
+
+export async function bandClip2() {
+  const data = [
+    {Date: new Date("2022-12-01"), Count: 10},
+    {Date: new Date("2022-12-02"), Count: 1},
+    {Date: new Date("2022-12-02"), Count: 1},
+    {Date: new Date("2022-12-03"), Count: 2},
+    {Date: new Date("2022-12-04"), Count: 3},
+    {Date: new Date("2022-12-05"), Count: 4},
+    {Date: new Date("2022-12-06"), Count: 5}
+  ];
+  return Plot.plot({
+    grid: true,
+    x: {interval: d3.utcDay},
+    marks: [
+      Plot.ruleY([0]),
+      Plot.barY(data, Plot.groupX({y: "sum"}, {x: "Date", y: "Count", rx: 6, insetBottom: -3, clip: "frame"}))
```

Comment:
```suggestion
      Plot.barY(data, Plot.groupX({y: "sum"}, {x: "Date", y: "Count", rx: 6, insetBottom: -6, clip: "frame"}))
```

To be clear I don't mind it being -3, but as an example it complicates things as it raises the unnecessary question "is this supposed to be half the rx?".